alphagov/paas-cf
# [#155592565] Bosh links phase 3

## What

Part 3, in which we remove the nats static IPs, adjust the static/dynamic ranges and start relying on implicit Bosh links to get addresses.

## How to review

- Ensure https://github.com/alphagov/paas-cf/pull/1278 is merged and deployed first
- Rebase this on master. There should be two commits left: a range adjustment and commit revert.
- Deploy ensuring that Bosh does nothing - it should 'choose' to use the existing nats IPs that are now in the dynamic range.

## Who can review

Not @camelpunch, @LeePorte or @bandesz.
